### PR TITLE
Fixing the Caught error

### PR DESCRIPTION
**Pull request by gautamkmr**
## Description ##
Fixing the Caught error

### Essentials ###
- [x] Changes are complete (i.e. I finished coding on this PR)

### Changes ###
- [x] Print actual error when error happens in Jenkinsfile

## Comments ##
- This change is on CI infra only.

### CODE REVIEW

**Comment by gautamkmr** [comment]
@szha @indhub

**Comment by szha** [comment]
Please maintain the checklist.

File: Jenkinsfile [hunk 1]
```diff
@@ -525,7 +525,7 @@ try {
   currentBuild.result = "SUCCESS"
 } catch (caughtError) {
     node("mxnetlinux") {
-        sh "echo caught error"
+        sh "echo caught ${caughtError}"
```
Comment (szha) [inline Jenkinsfile]: Why is this limited to `node("mxnetlinux")`? There are other nodes in the try block.

File: Jenkinsfile [hunk 2]
```diff
@@ -24,6 +24,7 @@ def init_git() {
     } catch (exc) {
       deleteDir()
       error "Failed to fetch source codes"
+      error "${exc}"
```
Comment (piiswrong) [inline Jenkinsfile]: combine into one error?

**Pull request merged**